x-pack/plugins/apm/public/context/pointer_event_context.tsx

```diff
@@ -12,21 +12,23 @@ import React, {
   useState,
 } from 'react';
 
-export const ChartsSyncContext = createContext<{
-  event: any;
-  setEvent: Dispatch<SetStateAction<{}>>;
+import { PointerEvent } from '@elastic/charts';
+
+export const PointerEventContext = createContext<{
+  pointerEvent: PointerEvent | null;
+  setPointerEvent: Dispatch<SetStateAction<PointerEvent | null>>;
 } | null>(null);
 
-export function ChartsSyncContextProvider({
+export function PointerEventContextProvider({
```

Comment:
I like the idea of changing the name, but if I open the context directory and see `pointer_event_context`, it still looks unclear to me for what this context is used. This is something specific for charts maybe calling it `chart_pointer_event_context` will make it clear for what it is used. WDYT?

Reply:
Sure, we can do that 👍